### PR TITLE
[improve][doc] Update links to the contribute guide

### PR DESCRIPTION
**Pull request by tisonkun**
### Motivation

This reflects https://github.com/apache/pulsar-site/pull/295.

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,21 +192,18 @@ Check https://pulsar.apache.org for documentation and examples.
...
 
 ```bash
 mvn clean install -DskipTests
 mvn package -Pdocker,-main -am -pl docker/pulsar-all -DskipTests
 ```
 
-After the images are built, they can be tagged and pushed to your custom repository.
-Here's an example of a bash script that tags the docker images with the current version and git revision and
-pushes them to `localhost:32000/apachepulsar`.
+After the images are built, they can be tagged and pushed to your custom repository. Here's an example of a bash script that tags the docker images with the current version and git revision and  pushes them to `localhost:32000/apachepulsar`.
```
Comment (momo-jun) [inline README.md:206]:
```suggestion
After the images are built, they can be tagged and pushed to your custom repository. Here's an example of a bash script that tags the docker images with the current version and git revision and pushes them to `localhost:32000/apachepulsar`.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,6 +21,4 @@
 
 ## Contributing to Apache Pulsar
 
-We would love for you to contribute to Apache Pulsar and make it even better!
-Please check the [Contributing to Apache Pulsar](https://pulsar.apache.org/contributing/)
-page before starting to work on the project.
+We would love for you to contribute to Apache Pulsar and make it even better! Please check the [Apache Pulsar Contributing Guide](https://pulsar.apache.org/contribute/) page before starting to work on the project.
```
Comment (momo-jun) [inline CONTRIBUTING.md:24]:
```suggestion
We would love for you to contribute to Apache Pulsar and make it even better! Please check the [Apache Pulsar Contributing Guide](https://pulsar.apache.org/contribute/) before starting to work on the project.
```

**Review by momo-jun — APPROVED** [review]
LGTM. Only left a couple of minor suggestions.

**Comment by tisonkun** [comment]
@momo-jun Thanks! Comments addressed.

**Comment by tisonkun** [comment]
Merging...

Review after merge is welcome.

**Pull request merged**